src/D2L.Bmx/Aws/AwsCredsCache.cs

```diff
@@ -69,6 +69,13 @@ AwsCredentials credentials
 		string jsonString = JsonSerializer.Serialize( prunedEntries.ToList(),
 			SourceGenerationContext.Default.ListAwsCacheModel );
 
+		if( !Directory.Exists( BmxPaths.CACHE_DIR ) ) {
+			Directory.CreateDirectory( BmxPaths.CACHE_DIR );
+			File.Delete( Path.Join( BmxPaths.BMX_DIR, "awsCredsCache" ) );
+			File.Delete( Path.Join( BmxPaths.BMX_DIR, "sessions" ) );
+			File.Delete( Path.Join( BmxPaths.BMX_DIR, "update_check" ) );
+		}
```

Comment:
Consider moving this up closer to the application entry point (the print/write/login handlers?)

Part of the direction we want to move here is to have less duplicated code that has to concern itself about the status of the  files in ``.bmx``